src/main/java/net/atos/entng/rss/model/Channel.java

```diff
@@ -0,0 +1,77 @@
+package net.atos.entng.rss.model;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import net.atos.entng.rss.constants.Field;
+import net.atos.entng.rss.helpers.IModelHelper;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Channel implements IModel<Channel> {
+    private String _id;
+    private JsonObject created;
+    private JsonObject modified;
+    private List<ChannelFeed> feeds;
+    private JsonObject owner;
+    private boolean global;
+    public Channel () {
+    }
+    public Channel(JsonObject channel) {
+        this._id = channel.getString(Field.MONGO_ID, null);
+        this.created = channel.getJsonObject(Field.CREATED, null);
+        this.modified = channel.getJsonObject(Field.MODIFIED, null);
+        this.owner = channel.getJsonObject(Field.OWNER, null);
+        this.global = channel.getBoolean(Field.GLOBAL, false);
+        this.feeds = IModelHelper.toList(channel.getJsonArray(Field.FEEDS, new JsonArray()), ChannelFeed.class);
+    }
+    public String getId() {
+        return _id;
+    }
+    public void setId(String id) {
+        this._id = id;
+    }
```

Comment:
Sur tes set, return this plutôt, ça permet d'en chainer plusieurs si on veut définir des valeurs comme ça

```suggestion
    public Channel setId(String id) {
        this._id = id;
        return this;
    }
```